node/middlewares/getSkuById.ts

```diff
@@ -1,13 +1,34 @@
 export async function getSkuById(ctx: Context, next: () => Promise<any>) {
   const {
     clients: { returnApp: returnAppClient },
+    vtex: { logger },
   } = ctx
 
   const { id } = ctx.vtex.route.params
-  const response = await returnAppClient.getSkuById(ctx, id)
 
-  ctx.status = 200
-  ctx.body = response
+  try {
+    const response = await returnAppClient.getSkuById(ctx, id)
+
+    if (!response) {
+      throw new Error(`Error getting SkuById for id: ${id}`)
+    }
+
+    logger.info({
+      message: 'Get SKU by id successfully',
+      data: response,
+    })
+
+    ctx.status = 200
+    ctx.body = response
+  } catch (e) {
+    logger.error({
```

Comment:
Wouldn't having this catcher here defeat the purpose of the `errorHandler` wrapper?
I mean, the `throw` in line 13 would be catched by this try/catch, rendering the errorHandler catcher redundant.

Reply:
> Looks good! I would only change how we handle the error in `getSkuById`
 jaja yes! I forgot to update that one